packages/angular/src/date-time-picker/date-time-picker.component.ts

```diff
@@ -48,6 +48,7 @@ export type DateRange = [Date, Date];
         (keydown.space)="togglePicker()"
         [ibmTooltip]="formatCurrentRange()"
         [offset]="tooltipOffset"
+        [disabled]="!formatCurrentRangeTitle()"
```

Comment:
Good thinking, though if we need to disable this we should probably have a separate `disabled` input we can pass down